en/docs/deploy-and-publish/deploy-on-gateway/api-gateway/gateway-policies.md

```diff
@@ -0,0 +1,98 @@
+#Gateway Policies
+
+In WSO2 API Manager, managing Global Level Policies within the gateway infrastructure is made easier. This streamlines policy handling, eliminating the need for administrators to manually create XML-type policy files. This feature facilitates a user-friendly approach to handling policies, eliminates the requirement for manual XML creation and placement in specific directories, and enhances control by providing a clear interface for creating, deploying, and undeploying policies, offering administrators better control over policy management.
+
+!!! note
+    Policies added globally using the older approach: creating XML files following the naming pattern `WSO2AM--Ext--<DIRECTION>` and saving them in the `<API-M_HOME>/repository/deployment/server/synapse-configs/default/sequences` directory, will not be overridden by the global policies added through the UI. Initially, the system will execute the policies added via manual files and then proceed to execute the policies deployed from the UI.
```

Comment:
> Initially, the system will execute the policies added via manual files and then proceed to execute the policies deployed from the UI.

The above can be reworded as `Policies applied via manual file addition will take precedence over the policies deployed through the UI`, if that makes sense. Shall we use the wording `take precedence over` based on the correct ordering here?

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7526/commits/64bafa0aa418ed0dbbc05491694766f9d24f000b.
Thank you for your suggestion.